### ISSUE TITLE
New apps are required to target API level 30 (Android 11) and use the Android App Bundle publishing format.

### ISSUE THREAD

**Issue by pazkero** [renpy/renpy-build#33]
Starting **August 2021**, new apps will be required to target API level 30 (Android 11) and use the Android App Bundle publishing format. Starting November 2021, all app updates will be required to target API level 30 (Android 11). Apps with a download size of more than 150 MB are now supported by Play Asset Delivery and Play Feature Delivery.

Learn more at: https://android-developers.googleblog.com/2020/11/new-android-app-bundle-and-target-api.html
See also: https://developer.android.com/distribute/best-practices/develop/target-sdk
See also: https://developer.android.com/about/versions/11/behavior-changes-11

![Google Play Console Screenshot](https://user-images.githubusercontent.com/8108358/128620429-c25d9641-c489-44a2-82be-db08115d95b6.png)

----
**TL;DR** RAPT is no longer able to upload apps to Google Play Store. Support to _update_ them will be dropped in November.
